### PR TITLE
Fixes #2333 Adds Russian translations

### LINKED ISSUE #2333: Translate App Strings to Russian
### User Story

As a Russian language speaker, I want to be able to interact with Status in my native language.

### Description

*Type*: Feature

*Summary*: We need someone fluent (and eloquent) in Russian to translate all the latest strings in Status from English. We would like to invite our community members to take on these tasks in order
a) To become a bit more acquainted with the Status repos and perhaps wander further into the codebase and set of issues we need help with, earning yourself Open Bounties all the way down this rabbit hole, and
b) To make sure that the product we deliver has been built by the same community that uses it, in a virtuous feedback loop of increasing understanding and value-generation.

#### Expected behavior
The Russian translation ought to be as up-to-date as the English one.

#### Actual behavior
We have not yet updated it, preferring to leave this open as an opportunity for our community to get involved again.

### Solution

*Summary*: 

- The file [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/ru.cljs) needs to be updated to match the new English one [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/en.cljs).
- How you translate is a matter of your preference. There were significant updates to the English file for the 0.9.12 release, with lots of strings both add and REMOVED. We need you to make sure that the files are exact mirrors of each other, except in different languages.
- It might be easier to simply take the new English file and translate from scratch rather than going through the current translated files line by line (though we leave this up to you).

### Additional Information

* Status version: 0.9.12+

There are bounties available for these translations. Please see the QR codes that will be added below after the DevCon 3 announcement.

### PR DESCRIPTION
**Pull request by dega1989**
Fixes #2333

Making it more native for Russian speaking persons!

### CODE REVIEW

**Review by annadanchenko — CHANGES_REQUESTED** [review]
"Извените" is not correct. Please, re-check spelling and grammar

File: src/status_im/translations/ru.cljs [hunk 1]
```diff
@@ -2,308 +2,363 @@
...
+   :connected                             "Подключено"
+   :process-json                          "Обработка JSON"
+   :error-processing-json                 "Огибка обработки JSON"
+   :rpc-url                               "RPC ссылка"
+   :remove-network                        "Убрать сеть"
+   :network-settings                      "Настройки сети"
+   :edit-network-warning                  "Бкдьте осторожны, изменение данных сети может отключить вас от нее"
+   :connecting-requires-login             "Для подключение к другой сети необходимо авторизироваться"
+   :close-app-title                       "Острожно!"
+   :close-app-content                     "Приложение будет остановленно и закрыто. Когда вы его снова откроете, булкт использована выбранная сеть"
```
Comment (rasom) [inline src/status_im/translations/ru.cljs]: `булкт`

**Review by dega1989 — COMMENTED** [review]
Changed "Извените" for "Извините".
str №114

**Comment by annadanchenko** [comment]
@dega1989 please re-check spelling and grammar for all of your translations. There are too many typos, I can't list them all

**Comment by dega1989** [comment]
Update ru.cljs
Str №18, 87, 114, 155, 158, 174, 209, 271, 275, 280, 284, 284, 290, 300, 319, 325, 326, 327, 360, 361, 363.

**Comment by annadanchenko** [comment]
Typos still survived after the spell check. For example "Количестро", "Сслылка". I didn't run the full check as it looks like what this PR is about - translate it in a nice way, so Russian-speaking audience can enjoy the app in Russian.

**Review by dega1989 — COMMENTED** [review]
Check This one!

**Comment by rasom** [comment]
closing this PR because https://github.com/status-im/status-react/pull/2361 was already merged

**Pull request closed without merging**